ci/travis_script_java.sh

```diff
@@ -27,7 +27,7 @@ if [ $ARROW_TRAVIS_JAVA_BUILD_ONLY == "1" ]; then
     # Save time and make build less verbose by skipping tests and style checks
     mvn -DskipTests=true -Dcheckstyle.skip=true -B install
 else
-    mvn -B install
+    mvn -B install -Dlogback.configurationFile=file:${ARROW_HOME}/java/dev/logback.xml
```

Comment:
It doesn't look like it's able to find the logback file. You might try `$JAVA_DIR/dev/logback.xml`